### DISCUSSION TITLE
Install things in python's site-packages path

### DISCUSSION THREAD

**Discussion by kpeeters** [Homebrew/discussions#5297] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

I am maintaining the tap `kpeeters/repo` which contains `cadabra2`. This is a complicated package which contains C++ code but also python packages. These python packages get installated (when doing a normal `make install` outside of homebrew) in subdirectories of the python `site-packages` path.

Homebrew seems to prevent me from writing into `site-packages` with an error of the form
```
  file cannot create directory:
  /opt/homebrew/opt/python@3.12/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/cadabra2_jupyter.
  Maybe need administrative privileges.
```
What should I be doing differently in order to be able to install this python library code? I have read python-for-formula-authors but I am not sure the above case is covered.

Thanks!

**Comment by ZhongRuoyu** [comment, marked as answer]
They should instead be installed to `prefix/Language::Python.site_packages("python3.12")`. The exact method to install there depends on the formula, but you may have a look at [`tbb.rb`](https://github.com/Homebrew/homebrew-core/blob/c02877e206c5d898c071a1cca4f3b013a5c31f39/Formula/t/tbb.rb), [`openvino.rb`](https://github.com/Homebrew/homebrew-core/blob/c02877e206c5d898c071a1cca4f3b013a5c31f39/Formula/o/openvino.rb), and many other formulae in [`homebrew/core`](https://github.com/Homebrew/homebrew-core).

**Reply by kpeeters** [reply]
Thanks, that did the trick.
